cheroot/test/test_core.py

```diff
@@ -17,20 +17,24 @@
 
 
 localDir = os.path.dirname(__file__)
-base_path = os.path.join(os.getcwd(), localDir)
-favicon_path = os.path.join(base_path, '../favicon.ico')
+temp_dir = os.path.join(os.getcwd(), localDir, 'temp')
```

Comment:
Well I think `temp` is a more suitable name consider its usage. And I try run `pytest` on the whole test suite, which turns out to be all passed.

```
➜  test git:(cheroot-fixbug-auto-create-static-file) ✗ pytest
================= test session starts ===========================
platform darwin -- Python 3.5.1, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /Users/rim99/PycharmProjects/baige, inifile: pytest.ini
collected 56 items 

test_compat.py s.
test_config_server.py ssss
test_conn.py sssssssssssssssss
test_core.py s..................
test_http.py .s.......
test_wsgiapps.py .....

============ 32 passed, 24 skipped in 92.70 seconds ===============

```